packages/hardhat-waffle/package.json

```diff
@@ -33,14 +33,14 @@
     "@types/chai": "^4.2.0",
     "@types/fs-extra": "^5.1.0",
     "chai": "^4.2.0",
-    "ethereum-waffle": "^3.0.0",
-    "ethers": "^5.0.0",
+    "ethereum-waffle": "^2.3.0",
+    "ethers": "^4.0.0",
     "hardhat": "^0.1.0-rc.0"
   },
   "peerDependencies": {
     "@nomiclabs/hardhat-ethers": "^0.1.0-rc.0",
-    "ethers": "^5.0.0",
-    "ethereum-waffle": "^3.0.0",
+    "ethers": "^4.0.0",
+    "ethereum-waffle": "^2.3.0",
```

Comment:
This plugin has a lot of imported things from waffle. Did you check that they are in sync with this version?

Reply:
By imported I mean copy-pasted code.

Reply:
Hm, let me check.